ranger/config/commands.py

```diff
@@ -838,12 +838,17 @@ class mkdir(Command):
     def execute(self):
         from os.path import join, expanduser, lexists
         from os import makedirs
+        from ranger.container.file import File
 
         dirname = join(self.fm.thisdir.path, expanduser(self.rest(1)))
         if not lexists(dirname):
             makedirs(dirname)
+            dir_new = File(dirname)
```

Comment:
This doesn't work with `:mkdir x/y/z`. Perhaps use `self.fm.select_file(dirname)`, it's made for this :)